spec/routines/user_from_pre_auth_state_spec.rb

```diff
@@ -1,29 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe UserFromPreAuthState, type: :routine do
```

Comment:
If really deleting PreAuth states, then should delete the routine this spec is testing

Reply:
Deleting this routine is causing a somewhat cascading effect with other routines. I'm going to push the changes before I start messing with specs (which will likely be a mess)